test/test_run_db_test_external_db.py

```diff
@@ -13,10 +13,15 @@ def setUp(self):
         self.test_environment = exaslct_utils.ExaslctTestEnvironmentWithCleanUp(
             self, exaslct_utils.EXASLCT_DEFAULT_BIN
         )
+        self.itde_test_environment = exaslct_utils.ExaslctTestEnvironmentWithCleanUp(
```

Comment:
In theory, we could also replace it by the API, because the itde cli is not what we test

Reply:
But it's easier to use the ExaslctTestEnvironmentWithCleanUp() as it prepares the parameter for `spawn_test_environment()` with the common values.

Reply:
I meant the API Version of the Environment

Reply:
Done